interfaces/apparmor/spec.go

```diff
@@ -143,7 +143,13 @@ func (spec *Specification) UpdateNS() map[string][]string {
 
 func snippetFromLayout(layout *snap.Layout) string {
 	mountPoint := layout.Snap.ExpandSnapVariables(layout.Path)
-	return fmt.Sprintf("# Layout path: %[1]s\n%[1]s{,/**} mrwklix,", mountPoint)
+	if layout.Bind != "" || layout.Type == "tmpfs" {
+		return fmt.Sprintf("# Layout path: %s\n%s{,/**} mrwklix,", mountPoint, mountPoint)
+	} else if layout.BindFile != "" {
+		return fmt.Sprintf("# Layout path: %s\n%s mrwklix,", mountPoint, mountPoint)
+	}
+	return fmt.Sprintf("# Layout path: %s\n# (no extra permissions necessary)", mountPoint)
```

Comment:
Interesting. I like that you aren't just giving the symlink perms away. Maybe mention this is a symlink? no extra permissions required for symlink?

Reply:
Sure, I'll do that.